opaque_keys/edx/django/tests/test_models.py

```diff
@@ -90,6 +90,10 @@ def test_fetch_from_db(self):
         fetched = ComplexModel.objects.filter(course_key=self.course_key).first()
         self.assertEqual(fetched, self.model)
 
+    def test_fetch_from_db_with_str(self):
+        fetched = ComplexModel.objects.filter(course_key=str(self.course_key)).first()
```

Comment:
Nit: can you use six.text_type() rather than str() here?  It shouldn't change anything, but we're trying to normalize this to help migrate to python3.

Reply:
@pwnage101 Done and also squashed commits.